ci-operator/step-registry/p2p/mtv-execute-bidir-live-migration/p2p-mtv-execute-bidir-live-migration-commands.sh

```
@@ -0,0 +1,369 @@
+#
+set -euxo pipefail; shopt -s inherit_errexit
+
+eval "$(
+    typeset -a fURLArr=()
+    type -t wget 1>/dev/null && fURLArr=(wget -nv -O-) || fURLArr=(curl -fsSL)
+    "${fURLArr[@]}" https://raw.githubusercontent.com/RedHatQE/OpenShift-LP-QE--Tools/refs/heads/main/libs/bash/common/EnsureReqs.sh
+)"; EnsureReqs jq
+
+if [[ -n "${SHARED_DIR}" && -s "${SHARED_DIR}/proxy-conf.sh" ]]; then
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/proxy-conf.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./proxy-conf.sh was not specified as input (see shellcheck -x).

(SC1091)